src/matrix/ssss/recoveryKey.ts

```diff
@@ -13,17 +13,20 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import {Key} from "./common.js";
+import {Key} from "./common";
+import {KeyDescription} from "./common";
+import type {Platform} from "../../platform/web/Platform.js";
+import type * as Olm from "@matrix-org/olm"
```

Comment:
How about this to avoid the typeof everywhere?
```suggestion
import type * as OlmNamespace from "@matrix-org/olm";
type Olm = typeof OlmNamespace;
```